Jack/Script.fsx

```diff
@@ -1,12 +1,15 @@
-﻿#r "../packages/FSharpx.Collections/lib/net40/FSharpx.Collections.dll"
-
-#load "Numeric.fs"
-#load "Seed.fs"
-#load "Tree.fs"
-#load "Shrink.fs"
-#load "Random.fs"
-#load "Gen.fs"
-#load "Property.fs"
+﻿#if INTERACTIVE
```

Comment:
I think maybe this `#if INTERACTIVE` isn't required? Just loading the files (instead of the assembly) seems to work on my MacBook with Mono.